src/components/filter/filter.tsx

```diff
@@ -25,15 +25,15 @@ export const Filter: React.FC<FilterProps> = ({ onFilterChange }) => {
 
 	const paddingMap = {
 		[TreeAgeIntervalIdentifier.Young]: {
-			padding: "p-3 lg:p-4",
+			size: "w-10 h-10",
 			label: i18n.filter.youngTrees,
 		},
 		[TreeAgeIntervalIdentifier.Medium]: {
-			padding: "p-2 lg:p-3",
+			size: "w-12 h-12",
 			label: i18n.filter.mediumTrees,
 		},
 		[TreeAgeIntervalIdentifier.Old]: {
-			padding: "p-1 lg:p-2",
+			size: "w-14 h-14",
 			label: i18n.filter.oldTrees,
```

Comment:
I checked again on figma and the trees should be scaled a bit smaller I tried it out briefly and I think these sizes work well: 

	[TreeAgeIntervalIdentifier.Young]: {
			size: "w-6 h-6",
			label: i18n.filter.youngTrees,
		},
		[TreeAgeIntervalIdentifier.Medium]: {
			size: "w-8 h-8",
			label: i18n.filter.mediumTrees,
		},
		[TreeAgeIntervalIdentifier.Old]: {
			size: "w-10 h-10",
			label: i18n.filter.oldTrees,
		},